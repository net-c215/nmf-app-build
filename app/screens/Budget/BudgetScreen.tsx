import React from "react";
import { ScrollView } from "react-native";

import styles from "./BudgetScreen.styles";

export default function BudgetScreen(): React.ReactElement {
  return <ScrollView style={styles.container}></ScrollView>;
}
