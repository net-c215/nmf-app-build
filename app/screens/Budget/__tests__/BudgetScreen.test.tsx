import React from "react";
import renderer from "react-test-renderer";
import BudgetScreen from "../BudgetScreen";

const props = {
  navigation: {
    push: () => {
      // do nothing.
    }
  }
};

it("BudgetScreen renders correctly", () => {
  const tree = renderer.create(<BudgetScreen {...props} />).toJSON();
  expect(tree).toMatchSnapshot();
});

// TODO: add test to verify that monthly budget is opened
