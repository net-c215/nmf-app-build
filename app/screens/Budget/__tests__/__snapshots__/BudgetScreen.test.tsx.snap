// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BudgetScreen renders correctly 1`] = `
<RCTScrollView
  style={
    Object {
      "backgroundColor": "#FFFFFF",
      "flex": 1,
      "paddingHorizontal": 16,
    }
  }
>
  <View>
    <ProgressChart
      isMonth={true}
    />
    <Button.Primary
      fullWidth={true}
      onPress={[Function]}
      style={
        Object {
          "marginVertical": 6,
        }
      }
      textType="Primary"
    >
      <Text.Primary
        bold={true}
        center={true}
        white={true}
      >
        Set monthly budget
      </Text.Primary>
    </Button.Primary>
    <NumberOfDaysVegetarian />
    <ProgressChart />
  </View>
</RCTScrollView>
`;
